### PR TITLE
Add team section

### PR DESCRIPTION
**Pull request by feliciofilipe**

### CODE REVIEW

**Review by GresosJ — COMMENTED** [review]
Pls run the formatter

File: src/data/team.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        "role": "Product Lead at Cyberdyne Systems",
+        "image": "erhart-cockrin.jpg"
+      },
+      {
+        "name": "Parker Johnson",
+        "role": "UI Designer at MomCorp",
+        "image": "parker-johnson.jpg"
+      }
+    ]
+  }
```
Comment (pedrofp4444) [inline src/data/team.json:121]: Please add "Programa" team :pray:

**Review by pedrofp4444 — CHANGES_REQUESTED** [review]

**Comment by pedrofp4444** [comment]
Please note that mobile responsiveness is not allowing to browse all existing teams :pray: 
felicio

File: src/data/team.json (hunk 1 above)
Comment (feliciofilipe) [reply src/data/team.json:121]: This is data collection its just a temporary placeholder for the `dev` environment. Once every detail is known and collected, a new PR should be open with those changes.

**Comment by feliciofilipe** [comment]
@pedrofp4444 I was not aware that the picker was so poorly done and did not scale to all number of different options. I fixed it on this component.

Everyone please be aware to perform this changes on other components if that change is posed.

**Review by pedrofp4444 — APPROVED** [review]

File: src/data/team.json (hunk 1 above)
Comment (pedrofp4444) [reply src/data/team.json:121]: Right, thank you :+1:

**Pull request merged**